images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -266,12 +266,15 @@ Get-DotnetRuntimes | Foreach-Object {
 }
 
 $markdown += New-MDHeader ".NET Framework" -Level 3
-$frameworks = Get-DotnetFrameworkTools
 $markdown += "``Type: Developer Pack``"
 $markdown += New-MDNewLine
-$markdown += "``Location $($frameworks.Path)``"
-$markdown += New-MDNewLine
-$markdown += New-MDList -Lines $frameworks.Versions -Style Unordered
+Get-DotnetFrameworkTools | Foreach-Object {
```

Comment:
Could you please create a separate function for this code?

`$markdown += Get-DotnetFrameworkToolsSection`